src/explorer/schemaTreeItem.ts

```diff
@@ -212,7 +212,7 @@ export default class SchemaTreeItem extends vscode.TreeItem
     this.cacheIsUpToDate = true;
 
     if (!schema || !schema.fields || schema.fields.length < 1) {
-      vscode.window.showInformationMessage(
+      void vscode.window.showInformationMessage(
```

Comment:
what would we wait with `await vscode.window.showInformationMessage`? Is it waiting for the info to be created or to be closed?

Reply:
This is the same as we talked here above about `showErrorMessage`. It waits when the modal is closed.